app/questionnaire/dynamic_answer_options.py

```diff
@@ -0,0 +1,34 @@
+from dataclasses import dataclass
+from typing import Any, Mapping
+
+from app.questionnaire.rules.operator import Operator
+from app.questionnaire.rules.rule_evaluator import RuleEvaluator
+
+
+@dataclass
+class DynamicAnswerOptions:
+    dynamic_options_schema: Mapping[str, Any]
+    rule_evaluator: RuleEvaluator
+
+    def evaluate(self) -> tuple[dict[str, str], ...]:
+        values = self.dynamic_options_schema["values"]
+        if "source" in values:  # pylint: disable=no-else-raise
+            # :TODO: Implement value sources support
```

Comment:
Never a fan of TODO, rather have a story

Reply:
We do have a story, but I think it's fine personally, as it gives an indication to the next dev where to start. Also, most IDE's support `TODO` and a good way to seen what's missing. Not precious about it, but I do think it is fine.